src/OrchardCore.Modules/OrchardCore.Title/Models/TitlePartSettings.cs

```diff
@@ -22,5 +22,10 @@ public class TitlePartSettings
         /// The pattern used to build the Title.
         /// </summary>
         public string Pattern { get; set; }
+
+        /// <summary>
+        /// Render TitlePart.
+        /// </summary>
+        public bool RenderTitle { get; set; }
```

Comment:
Should be true by default for backward compatibilty.

Reply:
I guess default should be what we want to keep going forward. Is it possible to use migration here for backward compatibility?

Reply:
I tried with `public bool RenderTitle { get; set; } = true;` but migration is not working